src/NuGetPackageVerifier/CompositeRules/PackageDependenciesCompositeRule.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGetPackageVerifier.Rules
+{
+    public class PackageDependenciesCompositeRule : IPackageVerifierRule
```

Comment:
This rule needs to specifically run *after* we do the timestamp/notimestamp split, so I'm pretty sure it needs to go into a new composite rule. However, I think the new composite rule should still have a more generic name, such as `NoTimestampPackagesCompositeRule` or something.